### PR TITLE
chore(suite): git ignore vscode settings

### PR DESCRIPTION
**Pull request by PeKne**
`.vscode/settings.json` git ignored. Allow creating customized workspace settings.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,7 +17,6 @@ node_modules
 .vscode/**/*
 !.vscode/launch.json
 !.vscode/tasks.json
```
Comment (Nodonisko) [inline .gitignore]: Can we remove also `tasks` and `launch` @matejkriz ?

**Review by Nodonisko — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (matejkriz) [reply .gitignore]: yes please, remove it

Comment (PeKne) [reply .gitignore]: changed with a313dafa7

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,36 +0,0 @@
-{
```
Comment (Nodonisko) [inline .vscode/settings.json:1]: maybe don't delete this file? because everyone will need to copy&paste again. I would prefer only ignore it.

**Comment by matejkriz** [comment]
P.S.: VS Code just added Profiles in January Update https://code.visualstudio.com/updates/v1_75#_profiles

File: .vscode/settings.json (hunk 2 above)
Comment (PeKne) [reply .vscode/settings.json:1]: Unfortunately, it is not possible. If you want to ignore a file, it has to be removed first. As described here: https://stackoverflow.com/a/25436481/8332763

**Review by Nodonisko — APPROVED** [review]

**Pull request merged**